src/css/custom.css

```diff
@@ -241,3 +241,24 @@ td {
     display: block !important;
   }
 }
+
+.tabs {
+  padding-left: 0;
+}
+
+.tabs__item {
+  border-bottom-color: var(--ifm-tabs-color-inactive-border);
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+  margin-right: 4px;
+  }
+  .tabs__item--active {
+  border-bottom-color: var(--ifm-tabs-color-active-border);
+  }
+  .tabs__item:hover {
+  background-color: var(--ifm-hover-overlay);
+  border-top-left-radius: 8px;
```

Comment:
Are you sure you want border radius on hover? Seems abnormal to me to change the radius on something when a user hovers. If so, you can simplify this as well by doing

`border-radius: 8px 8px 0 0;` (args go clockwise starting w/ top left)

Reply:
I think just keeping `border-radius: 0` and not changing it on hover is best.